notebooks/20220114_look_at_sra_org_composition.ipynb

```diff
@@ -10,7 +10,18 @@
   },
```

Comment:
<p>Is the change in distribution here expected? Is it a feature of the new GTDB version or upstream data cleaning? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-microberna/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>upstream data cleaning! I changed the axes too from sqrt to not sqrt, which makes the difference look a lot more dramatic than it actually is.</p>